t/react_template/spec.yaml

```diff
@@ -18,25 +18,34 @@ desc: 'An example template using React library'
 inputs:
   - name: 'automation_service_account'
     desc: 'Automation Service Account (ex: [account_name]@[project_id].iam.gserviceaccount.com)'
+    default: ''
   - name: 'wif_provider'
     desc: 'Workload Identity Federation Provider (ex: projects/[project_id]/locations/global/workloadIdentityPools/[WIF_pool_name]/providers/[provider_name])'
+    default: ''
```

Comment:
I dunno about this, template rendering will "succeed" but produce broken GH workflows.

Reply:
When we added the workflows, now inputs for these are required for this template to render.

I think eventually extracting out the workflows/Dockerfiles as discussed in [go/vendor-abcxyz-apps](http://goto.google.com/vendor-abcxyz-apps) is the best move.

But for now, this is blocking the template from being able to be rendered locally, without the workflows. The env file just looks like:

```
AUTOMATION_SERVICE_ACCOUNT=
WIF_PROVIDER=
AR_REPOSITORY=
AR_LOCATION=
CR_SERVICE=
REGION=
PROJECT_ID=
```

with the default blanks. Maybe I can change the defaults to comments of the value hints in the desc for each input?

Reply:
I think this is a design problem that we can't just hack around this way. If we're going to include the workflows in the template, we should try hard to make them work. So I think we should either not have defaults or factor them out.

Reply:
Ok, then I think maybe I'll prioritize factoring them out first before continuing with this PR. What do you think?

Reply:
That sounds fine to me